README.md

```diff
@@ -64,7 +64,15 @@ The functions must be of type `IO ()`, `IO Value`, `Value -> IO Value`, `Value -
 
 ## Development
 
-Use [stack] to build.  
+Building from the repository requires initialization with the Duktape configuration script the first time (these files are included in the cabal distribution on Hackage). 
```

Comment:
Looks like it might be possible to use `Setup.hs` with [hooks](https://www.haskell.org/cabal/release/latest/doc/API/Cabal/Distribution-Simple.html#t:UserHooks) to run the configure script?

Reply:
I thought about that. It takes a non-trivial time to run the script so we don't want to do it as part of every build, and it isn't needed for installation from the sdist package. Maybe a `preConf` hook that checks to see if the files are already there, and runs the script if not, and a `postClean` that removes them if it can find `configure.py`. You'd have to remember to clean after updating the Duktape version though. 

Alternatively `configure.py` also produces a `json` metadata file; I could look for that file and compare its version to what I find in the `duktape` repo, and always rebuild if they differ.

Reply:
I think I'd check if the modification time of the duktape source files is after the modification time of the files produced by configure.